object/permission.go

```diff
@@ -328,7 +328,7 @@ func DeletePermission(permission *Permission) (bool, error) {
 
 func getPermissionsByUser(userId string) ([]*Permission, error) {
 	permissions := []*Permission{}
-	err := ormer.Engine.Where("users like ?", "%"+userId+"\"%").Find(&permissions)
+	err := ormer.Engine.Where("users like ? OR JSON_CONTAINS(users, '\"*\"', '$')", "%"+userId+"\"%").Find(&permissions)
```

Comment:
will JSON_CONTAINS function work with other database?

Reply:
Yes, this is also my concern. That's obviously not standard SQL.

@Sesota you'd better create issue to discuss first, before making PR.